### DISCUSSION TITLE
Help with code

### DISCUSSION THREAD

**Discussion by DREWCOIN7** [community/community#22858] [New to GitHub]
<p>Hello,<br>
I need help downloading the complete algorithm on the link below. What do I click? Do I have to sign up first?</p>
<p><a href="https://github.com/ethereum-mining/ethminer" rel="nofollow noopener">https://github.com/ethereum-mining/ethminer</a></p>
<p>Thanks,<br>
Andrew</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/drewcoin7">@drewcoin7</a>,</p>
<p>Since GitHub doesn’t own or support Ethminer and just hosts its Git repository, I won’t be able to give you any in-depth help since I’m not familiar with the program. However, it looks like you can click one of the download buttons in the <a href="https://github.com/ethereum-mining/ethminer#install" rel="nofollow noopener">Install section of the README</a> to download and install the program. The README also seems to have information on what to do after you install the program and where you can go for help if you need it.</p>
<p>Hope that helps get you started!</p>
